types/src/front/lib/assistant.ts

```diff
@@ -11,6 +11,8 @@ export const MODEL_PROVIDER_IDS = [
   "mistral",
   "google_ai_studio",
 ] as const;
+export const DEFAULT_EMBEDDING_PROVIDER_ID = "openai";
```

Comment:
Embedding for now is tight to a data source, I'll suggest to move this to a file related to embeddings.